cli/tests/containerize.bats

```diff
@@ -112,18 +112,12 @@ function skip_if_linux() {
 }
 
 # bats test_tags=containerize:default-to-file
-@test "container is written to a file by default" {
+@test "container is written to a runtime by default" {
```

Comment:
It would still be useful to have a test that we can write to a file in the absence of Podman or Docker in `PATH`, since the integration tests are our only real coverage for most of `flox containerize` and everything the user cares about is the "integration" part.

Happy to forgo Docker though on the basis that it might be a pain to setup in CI and we have some certainty about the container image format being the same for Podman.